devenv.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+
+wait_for() {
+    echo "waiting for $1"
+
+    for _ in $(seq 10) ; do
+      if pg_isready -d postgres -h localhost -p "$1" -U postgres 1>/dev/null 2>&1; then
+        echo "connector up"
+        return 0
+      fi
+      echo -n "."
+      sleep 1
+    done
+    echo
+    echo "FAIL waiting for $1"
+    exit 1
+}
+
+cargo pgx start
+wait_for 28814
+psql -h localhost -p 28814 -d template1 -c 'CREATE EXTENSION IF NOT EXISTS timescaledb';
+createdb -h localhost -p 28814 "$(whoami)"
+# This allows entr to work correctly on docker for mac
+export ENTR_INOTIFY_WORKAROUND=true
+
+# Note: this is not a comprehensive list of source files
+SOURCE_FILES="src migration"
+find ${SOURCE_FILES} | entr -n cargo pgx install > ~/compile.log &
```

Comment:
Should we run `make package` here?
```suggestion
find ${SOURCE_FILES} | entr -n make package > ~/compile.log &
```

Reply:
`cargo pgx install` puts the binaries into the correct pg instance's lib directory, while `make package` merely produces them inside the `target` directory. Plus `make package` performs a couple of extra actions that would slow down development.

Reply:
Am I missing something here? `cargo pgx install` also compiles & installs our extension?

Reply:
Looks like it does, ignore this comment!